kf/bpf.go

```diff
@@ -523,40 +523,79 @@ func (b *BPF) VerifyAndGetArtifacts(conf *config.Config) error {
 // GetArtifacts downloads artifacts from the nexus repo
 func (b *BPF) GetArtifacts(conf *config.Config) error {
 	var fPath = ""
+	buf := &bytes.Buffer{}
+	//      "file://" ---> has size 7
+	//     "http://" ---> has size 7
+	if len(b.Program.EPRUrl) >= 7 {
```

Comment:
We should just use url.parse to determine the scheme (http, https, file, ...).  File should have three slashes "file:///"